src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -120,6 +120,17 @@ public void testUnused() throws IOException {
     assertTrue(imports.contains("org.junit.Assert.*"));
   }
 
+  @Test
+  public void testUnused2() throws IOException {
+    Path p = Paths.get(System.getProperty("user.dir"), "src", "test", "resources",
+        "UnusedImports2.java");
+    Result result = new ImpSort(StandardCharsets.UTF_8, eclipseDefaults, true, true).parseFile(p);
+    Set<String> imports =
+        result.getImports().stream().map(Import::getImport).collect(Collectors.toSet());
+    assertEquals(2, imports.size());
+    assertEquals(2, result.getImports().size());
+  }
+
```

Comment:
Should combine this with the existing unused import test case, and verify that the contained imports are what is expected, rather than merely just check the count.

Reply:
I did this myself when I merged.